### PR TITLE
snakefile: move copy_default_files and process_run_config to helpers

### PR DESCRIPTION
**Pull request by FabianHofmann**
In the current master, the CI on windows fails due to some weird syntax error (probably something wrong in snakemake >= 8, but hard to track). Took the opportunity and moved some snippets from the Snakefile header into dedicated functions.

### CODE REVIEW

File: Snakefile [hunk 1]
```diff
@@ -2,44 +2,25 @@
 #
 # SPDX-License-Identifier: MIT
 
-from os.path import normpath, exists
-from shutil import copyfile, move, rmtree
```
Comment (fneum) [inline Snakefile:6]: needs to be imported in _helpers.py

Comment (FabianHofmann) [reply Snakefile:6]: it was the `move` function which is needed in some smk files (and the rmtree is used as well). Reverted this

**Comment by FabianHofmann** [comment]
annoying, it all works but does not resolve the original problem with the indentation error on windows machines... still we can merge this

**Comment by FabianHofmann** [comment]
I really don't know what's wrong with these windows machines... For a temporal fix, I removed windows from the CI requirement list. l

**Pull request merged**